CHANGELOG

```diff
@@ -1,3 +1,8 @@
+## rackspace-monitoring-agent-2.6.22
+
+* Upgrade to latest luvi version to pick up 2.9.3 and openssl upgrade
```

Comment:
```suggestion
* Upgrade to latest luvi version to pick up sigar 2.9.3 and openssl upgrade
```

Reply:
Oops, I'll fix that wording when I bump the version to non-beta